noir-projects/aztec-nr/aztec/src/state_vars.nr

```diff
@@ -1,7 +1,15 @@
-mod immutable_singleton;
 mod map;
-mod public_state;
+mod private_immutable;
+mod private_mutable;
+mod public_mutable;
 mod set;
-mod singleton;
-mod stable_public_state;
+mod shared_immutable;
 mod storage;
+
+use crate::state_vars::map::Map;
```

Comment:
With these, should be update the imports generally to make it nicer, might be fine to do in a separate pr so it don't bloat in here.

Reply:
All the occurrences are addressed now other than the one of Storage which for whatever reason fails to compile when trying to use the re-export. I guess it's caused by it being a trait and not struct.

Are you aware of this issue? If not will ask noir people.